kiwi/builder/live.py

```diff
@@ -241,8 +241,8 @@ def create(self):
         # calculate size and decide if we need UDF
         if rootsize.accumulate_mbyte_file_sizes() > 4096:
             log.info('ISO exceeds 4G size, using UDF filesystem')
-            custom_iso_args['create_options'].append('-allow-limited-size')
-            custom_iso_args['create_options'].append('-udf')
+            custom_iso_args['create_options'].append('-iso-level')
+            custom_iso_args['create_options'].append('3')
```

Comment:
No, it does not imply udf filesystem. As those flags are applied only on images bigger than 4Gb I just tested that iso-level 3 also supports such a big image, I have built it and dd'ed it to a USB, but I have not done it in real DVD, if that is the case you have in mind. Some days before I tested as CD-ROM input in a libvirt virtual machine.

Do you have in mind somewhere it could fail? I have no included this flag because is different in mkisofs (-UDF) and in genisoimage (-udf)